server.js

```diff
@@ -27,6 +15,23 @@ app.get('/', (req, res) => {
   res.send('Hello world')
 })
 
+app.get('/nominations', (req, res) => {
+  res.json(goldenGlobesDatadata)
+})
+
+app.get('/year/:year', (req, res) => {
```

Comment:
Following the RESTful rules, I think this endpoint should be a filter on the `/nominations` endpoint rather than an endpoint of its own, since they both return nominations.